Code/Framework/AzToolsFramework/Tests/ActionManager/ActionManagerTests.cpp

```diff
@@ -107,6 +107,108 @@ namespace UnitTest
         EXPECT_TRUE(actionTriggered);
     }
 
+    TEST_F(ActionManagerFixture, GetActionName)
+    {
+        m_actionManagerInterface->RegisterActionContext("", "o3de.context.test", {}, m_widget);
+        
+        AzToolsFramework::ActionProperties actionProperties;
+        actionProperties.m_name = "Test Name";
+        m_actionManagerInterface->RegisterAction("o3de.context.test", "o3de.action.test", actionProperties, []{});
+
+        auto outcome = m_actionManagerInterface->GetActionName("o3de.action.test");
+        EXPECT_TRUE(outcome.IsSuccess());
+        EXPECT_EQ(outcome.GetValue(), "Test Name");
+    }
+
+    TEST_F(ActionManagerFixture, SetActionName)
+    {
+        m_actionManagerInterface->RegisterActionContext("", "o3de.context.test", {}, m_widget);
+        
+        AzToolsFramework::ActionProperties actionProperties;
+        actionProperties.m_name = "Wrong Name";
+        m_actionManagerInterface->RegisterAction("o3de.context.test", "o3de.action.test", actionProperties, []{});
+
+        auto setOutcome = m_actionManagerInterface->SetActionName("o3de.action.test", "Correct Name");
+        EXPECT_TRUE(setOutcome.IsSuccess());
+
+        auto getOutcome = m_actionManagerInterface->GetActionName("o3de.action.test");
+        EXPECT_EQ(getOutcome.GetValue(), "Correct Name");
+    }
+
+    TEST_F(ActionManagerFixture, GetActionDescription)
+    {
+        m_actionManagerInterface->RegisterActionContext("", "o3de.context.test", {}, m_widget);
+        
+        AzToolsFramework::ActionProperties actionProperties;
+        actionProperties.m_description = "Test Description";
+        m_actionManagerInterface->RegisterAction("o3de.context.test", "o3de.action.test", actionProperties, []{});
+
+        auto outcome = m_actionManagerInterface->GetActionDescription("o3de.action.test");
+        EXPECT_TRUE(outcome.IsSuccess());
+        EXPECT_EQ(outcome.GetValue(), "Test Description");
+    }
+
+    TEST_F(ActionManagerFixture, SetActionDescription)
+    {
+        m_actionManagerInterface->RegisterActionContext("", "o3de.context.test", {}, m_widget);
```

Comment:
nit/suggestion: As this line appears in every test, it might be worthwhile creating a new lightweight fixture that includes this in Setup so the test only contains code relevant for the test and removes some duplication. This is super minor, but it's definitely useful to take a step back and review the tests every once in a while and see if there are any opportunities for refactoring

Reply:
Not really every test, but most of them for sure. I'll consider some light refactoring in follow-ups.